src/data/nat/parity.lean

```diff
@@ -235,6 +235,21 @@ by { rintro ⟨c, rfl⟩, simp [pow_mul] }
 theorem neg_one_pow_of_odd : odd n → (-1 : R) ^ n = -1 :=
 by { rintro ⟨c, rfl⟩, simp [pow_add, pow_mul] }
 
+lemma neg_one_pow_of_add_even {m n : ℕ} (hm : even m) :
```

Comment:
When I was testing this I had put the lemmas into the global namespace accidentally where the dot notation does work, but I see it doesn't because it's in the `nat` namespace. That's too bad.